internal/config/options.go

```diff
@@ -164,13 +180,12 @@ var defaultOptions = Options{
 		"Strict-Transport-Security": "max-age=31536000; includeSubDomains; preload",
 	},
 	Addr:                    ":443",
-	CertFile:                filepath.Join(fileutil.Getwd(), "cert.pem"),
```

Comment:
Not strongly. I'd rather keep backward compatibility. 

However, keeping iIt would change how options fail if `insecure_server` and no certs are provided in that it will always fail on trying to read a (potentially) non existent cert file.

e.g. if you run pomerium with no certs specified, no `cert.pem` in the root dir, and no insecure flag, will error with a fopen error on those non existent cert files. 

Which I'm okay with, but could be a bit confusing.

Reply:
Hm.  Yeah neither is ideal here.  Without considering prior defaults this is probably the right behavior - we'll just have to make sure it is addressed in usage elsewhere.  Add as a breaking change?

Reply:
I've changed the public interface to configuration options a bit to make this a bit more focused and without setting any unintended settings. Let me know what you think.

Reply:
👍